### PR TITLE
docs(readme): update events example

### PR DESCRIPTION
**Pull request by everdimension**
This PR contains a:

- [x] **metadata update**

### Motivation / Use-Case

`listening` event will no longer be emmited after the `serve()` promise is resolved: https://github.com/webpack-contrib/webpack-serve/releases/tag/v2.0.0

So I updated the events example with another event.

### Additional Info

In the API example I tried to make it clear that we're already *listening* by the time that the server promise is resolved

### CODE REVIEW

**Review by G-Rath — APPROVED** [review]
@shellscape this looks acceptable to me. 

The only other addition I could mention as maybe worth adding is just a line after the "When using the API directly" block, to the effect of "The `serve` function returns a Promise that resolves once the server starts listening'.

I think this does raise the question of should the listening event be documented in the first place, as is there actually any time where you can get a reference to `server` *before* it starts listening?

**Comment by everdimension** [comment]
@G-Rath 
> is there actually any time where you can get a reference to server before it starts listening?

I don't know about the reference to the `server`, but you can hook into the `listening` event using the [`options.on`](https://github.com/webpack-contrib/webpack-serve#optionson) property.

So I guess documenting the `listening` event still makes sense.

File: README.md [hunk 1]
```diff
@@ -390,7 +392,7 @@ const argv = {};
 const config = require('./webpack.config.js');
 
 serve(argv, { config }).then((server) => {
-  server.on('listening', ({ server, options }) => {
```
Comment (michael-ciniawsky) [inline README.md]: Why this change here ?

**Review by michael-ciniawsky — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (everdimension) [reply README.md]: I explained this both in the first comment and in the commit message:

> `listening` event will no longer be emmited after the serve() promise is resolved: https://github.com/webpack-contrib/webpack-serve/releases/tag/v2.0.0

I changed the example code to `'build-started'` event because it's the next best starting place for the reader to understand how to use the event hooks.

**Comment by alexander-akait** [comment]
DEPRECATED. Please use `webpack-dev-server` (he is in support and update mode again). Feel free to create a issue if some features are not implemented in `webpack-dev-server`. 

Why deprecated `webpack-serve` ?
- The author stopped developing the package.
- Two development servers are misleading for developers.
- Hard to maintain two packages with same purpose.
- Some `dependencies` are not available to fixes/implement new features/etc (https://www.npmjs.com/package/webpack-hot-client and https://github.com/shellscape/koa-static) and there's nothing we can do here.
- `webpack-dev-server` is stable.

Thanks for using `webpack`! We apologize for the inconvenience. In the future, we will avoid such situations.
